DEVELOPMENT.md

```diff
@@ -1,71 +1,195 @@
-export g_additionalModulePath="/path/to/deadline-cloud-for-cinema4d/deadline_cloud_extension"
+
+The equivalent run with the `cinema4d-openjd daemon` subcommand looks like:
+
+```bash
+# The daemon start command requires that the connection-info file not already exist.
+test -f connection-info.json || rm connection-info.json
+
+# This starts up a background process running the adaptor, and runs the `on_init` and `on_start`
+# methods of the adaptor.
+cinema4d-openjd daemon start \
+  --init-data file://<absolute-path-to-init-data.yaml> \
+  --connection-file file://connection-info.json
+
+# This connects to the already running adaptor, via the information in the connection-info.json file,
+# and runs the adaptor's `on_run` method.
+# When testing, we suggest doing multiple "daemon run" commands with different inputs before
+# running "daemon stop". This will help identify problems caused by data carrying over from a previous
+# run.
+cinema4d-openjd daemon run \
+  --run-data file://<absolute-path-to-run-data.yaml> \
+  --connection-file file://connection-info.json
+
+# This connects to the already running adaptor to instruct it to shutdown the Cinema 4D application
+# and then exit.
+cinema4d-openjd daemon stop \
+  --connection-file file://connection-info.json
 ```
 
-- run cinema4d
-- Extensions > Deadline Cloud Submitter
+#### Running the Adaptor on a Farm
+
+If you have made modifications to the adaptor and wish to test your modifications on a live Deadline Cloud Farm
+with real jobs, then we recommend using a [Service Managed Fleet](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/smf-manage.html)
+for your testing. We recommend performing this style of test if you have made any modifications that might interact with Deadline Cloud's
+job attachments feature, or that could interact with path mapping in any way. 
 
-## Worker adaptor environment
+You'll need to perform the following steps to substitute your build of the adaptor for the one in the service.
 
-Install deadline in a [supported search path](https://developers.maxon.net/docs/py/2024_0_0a/manuals/manual_py_libraries.html#python-interpreter-bound-search-paths) or set a [custom interpreter path](https://developers.maxon.net/docs/py/2024_0_0a/manuals/manual_py_libraries.html#custom-interpreter-bound-search-paths) to the deadline install location.
+1. Follow the [instructions](https://github.com/aws-deadline/deadline-cloud-samples/tree/mainline/conda_recipes#create-a-patch-for-a-recipe) to make a patch for your "cinema4d-openjd" recipe. 
+2. Build a new "cinema4d-openjd" conda package using this patch. For instructions on building conda recipes on Deadline Cloud, follow this [article](https://aws.amazon.com/blogs/media/create-a-conda-package-and-channel-for-aws-deadline-cloud/). 
+This should have automatically added the latest patch onto the S3 bucket for the SMF workers to pull from. 
+3. Submit jobs and check if the renders work as per expectations. 
```

Comment:
nit: `as per expectations` -> `as expected` (simpler)

Reply:
Sure, I'll update it.